backend/backend/views.py

```diff
@@ -12,11 +12,11 @@
 from libs.user_login_throttle import UserLoginThrottle
 from substrapp.orchestrator import get_orchestrator_client
 from substrapp.utils import get_owner
-from users.utils import bearer_token as bearer_token_utils
+from users.models.token import BearerToken
 
 
 def _bearer_token_dict(token: BearerToken, include_payload: bool = True) -> dict:
-    d = {"created_at": token.created, "expires_at": bearer_token_utils.expires_at(token)}
+    d = {"created_at": token.created, "expires_at": token.expires_at(), "note": token.note, "token_id": token.token_id}
     if include_payload:
         d["token"] = token.key
     return d
```

Comment:
This kind of behavior looks like the use-case of DRF serializer, (and how [DRF works internally for their tokens](https://github.com/encode/django-rest-framework/blob/master/rest_framework/authtoken/serializers.py)). Are we trying to make this part of the code less dependant of DRF (summoning @oleobal).

Reply:
done